scheduler/test/cook/test/mesos/api.clj

```diff
@@ -1140,6 +1140,7 @@
         (is (thrown? Exception (s/validate api/Job (assoc min-job :expected-runtime 3 :max-runtime 2))))))))
 
 (deftest test-create-jobs!
+  (cook.test.testutil/flush-caches!)
```

Comment:
Why are we `refer`ing `flush-caches!` but also specifying the namespace when we call it?